.env.sample

```
@@ -8,4 +8,7 @@ OSM_CONSUMER_SECRET=your-oauth-secre
 OSM_DOMAIN=https://www.openstreetmap.org
 SESSION_SECRET=super-secret-sessions
 DATABASE_URL=postgres://scoreboard:test@localhost:5433/scoreboard
-OSM_TEAMS_SERVICE=http://localhost:3000
\ No newline at end of file
+OSM_TEAMS_SERVICE=http://localhost:8989
+OSM_TEAMS_SERVICE_TOKEN_URL=http://localhost:4445
```

```suggestion
OSM_TEAMS_SERVICE_TOKEN_URL=http://localhost:4444
```

It looks like the port should be `4444` by default. At least that's what got things working for me

---

Yes you're correct